src/sentry/conf/server.py

```diff
@@ -2078,6 +2078,7 @@ def SOCIAL_AUTH_DEFAULT_USERNAME() -> str:
 
 SENTRY_METRICS_INDEXER_REINDEXED_INTS: dict[int, str] = {}
 
+LAST_SEEN_UPDATER_REINDEXED_ORGS: list[int] = []
```

Comment:
How do we intend to populate this list of orgs? Is it going to be the one customer org on which we have seen the issue?

Reply:
I think that was the idea, yes. We start with that one.